### DISCUSSION TITLE
Next.js 12 and Capacitor Routing Issues

### DISCUSSION THREAD

**Discussion by CyberCyclone** [vercel/next.js#36614] [Help]
I'm using Next.js 12 and Ionic Capacitor. Usually I'd put up a code example but the project I'm working on is to large to know where the issue could be.

My issue is that when navigating, the page just reloads the home page. E.g, if I went to the `/about` page, it stays on the home page but reloads it.

The default Next.js auto routing works when deployed to Vercel, however the issue is when it's wrapped by Capacitor.

If I navigate to `/about.html` the correct page loads, but not `/about`.

Is there any known things that I need to do to get routing to work or should it work out of the box?

**Comment by icyJoseph** [comment]
I wonder, what would happen if, for testing, you tell Next.js to use trailing slashes?
